doc/parameters/param_list.md

```diff
@@ -154,6 +154,8 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | Parameter | Default | Description |
 | ----------| --------| ------------|
 | controller_frequency | 20.0 | Frequency to run controller |
+| progress_checker_plugin | ["progress_checker"] | Check the progress of the robot |
```

Comment:
OK I think this all looks good, but the parameters need to move / be added to the nav2_bringup configuration yaml files (so moving the 2 plugin params into the right namespace, and adding the plugin parameter / `.plugin: type` field.

Reply:
Sorry I missed this one. Updated.